site/man/rabbitmq-streams.8.html

```diff
@@ -303,7 +314,155 @@ <h3 class="Ss" title="Ss" id="Stream_plugin"><a class="permalink" href="#Stream_
     <div class="Bd Bd-indent lang-bash"><code class="Li">rabbitmq-streams
       list_stream_publishers connection_pid publisher_id stream</code></div>
   </dd>
+  <dt><a class="permalink" href="#add_super_stream"><code class="Cm" title="Cm" id="add_super_stream">add_super_stream</code></a>
+    <var class="Ar" title="Ar">super-stream</var>
+    [<div class="Op"><code class="Fl" title="Fl">--vhost</code>
+    <var class="Ar" title="Ar">vhost</var></div>]
+    [<div class="Op"><code class="Fl" title="Fl">--partitions</code>
+    <var class="Ar" title="Ar">partitions</var></div>]
+    [<div class="Op"><code class="Fl" title="Fl">--routing-keys</code>
+    <var class="Ar" title="Ar">routing-keys</var></div>]
+    [<div class="Op"><code class="Fl" title="Fl">--max-length-bytes</code>
+    <var class="Ar" title="Ar">max-length-bytes</var></div>]
+    [<div class="Op"><code class="Fl" title="Fl">--max-age</code>
+    <var class="Ar" title="Ar">max-age</var></div>]
+    [<div class="Op"><code class="Fl" title="Fl">--stream-max-segment-size-bytes</code>
+    <var class="Ar" title="Ar">stream-max-segment-size-bytes</var></div>]
+    [<div class="Op"><code class="Fl" title="Fl">--leader-locator</code>
+    <var class="Ar" title="Ar">leader-locator</var></div>]
+    [<div class="Op"><code class="Fl" title="Fl">--initial-cluster-size</code>
+    <var class="Ar" title="Ar">initial-cluster-size</var></div>]</dt>
+  <dd>
+    <dl class="Bl-tag">
+      <dt><var class="Ar" title="Ar">super-stream</var></dt>
+      <dd>The name of the super stream to create.</dd>
+      <dt><var class="Ar" title="Ar">vhost</var></dt>
+      <dd>The name of the virtual host to create the super stream into.</dd>
+      <dt><var class="Ar" title="Ar">partitions</var></dt>
+      <dd>The number of partitions the super stream will have.</dd>
```

Comment:
How about changing this: **The number of partitions the super stream will have.** to this: **The number of partitions in the super stream.**